lte/gateway/python/magma/pipelined/tests/test_restart_resilience.py

```diff
@@ -687,6 +692,106 @@ def test_clean_restart(self):
         self.enforcement_controller._clean_restart = False
         self.enforcement_stats_controller._clean_restart = False
 
+    def test_with_stateless_enforcement_restart(self):
```

Comment:
@nstng , if we make 'redis_enabled': True under config section, then other test cases will fail with snapshot mismatch. hence, we are passing is_stateless member as a True to the stateless test case only so that existing test case snapshots will remain unchanged.

Reply:
Ah OK, I did no see that, sorry :see_no_evil: - by toggling `self.enforcement_controller.is_stateless` I can see now the test is failing with the removed flow/policy. :+1: